test/extended/registry/util/util.go

```diff
@@ -120,6 +120,19 @@ func GetRegistryPod(podsGetter kcoreclient.PodsGetter) (*kapiv1.Pod, error) {
 	return &podList.Items[0], nil
 }
 
+// LogRegistryPod attempts to write registry log to a file to recent test's output directory.
+func LogRegistryPod(oc *exutil.CLI) error {
+	pod, err := GetRegistryPod(oc.KubeClient().Core())
+	if err != nil {
+		return fmt.Errorf("failed to get registry pod: %v", err)
+	}
+	path, err := oc.Run("logs").Args("dc/docker-registry").OutputToFile("pod-" + pod.Name + ".log")
```

Comment:
@jim-minter if you want to change the Pod log dumping logic to put it in a file, this looks like a promising way to do it instead of spamming the stdout.

Reply:
@bparees yes, but currently it ends up in /tmp, not as an artifact.

Reply:
well that's not very useful...  and it makes me think @miminar should update this to dump to stdout then because this won't be accessible when the job is done, right?

 (or use one of the existing DumpPodLogs methods)

Reply:
+1, @miminar I don't think this output will be retrievable when run in the CI system?

Reply:
> +1, @miminar I don't think this output will be retrievable when run in the CI system?

I was happy using it locally. I didn't realize it won't be preserved in CI system.
Addressed by #16859